tests/testthat/test-precompile.R

```diff
@@ -0,0 +1,83 @@
+# Disable caching
+sass_cache_set(NULL)
```

Comment:
Can we do this locally (for this test script)?

Reply:
Yeah, I actually have this here just until we finalize and merge the sass caching changes.

Reply:
I just remembered: we also have stuff that disables caching in other test files, and those will have to be fixed as well.
https://github.com/rstudio/bootstraplib/blob/4788d871a9bb09a8f700e7cb4d886b161184054f/tests/testthat/test-bs4-deps.R#L4